src/System.Globalization/tests/CharUnicodeInfo/CharUnicodeInfoTests.netstandard1.7.cs

```diff
@@ -0,0 +1,111 @@
+            "\uaa50\uaa51\uaa52\uaa53\uaa54\uaa55\uaa56\uaa57\uaa58\uaa59" + 
+            "\uabf0\uabf1\uabf2\uabf3\uabf4\uabf5\uabf6\uabf7\uabf8\uabf9" + 
+            "\uff10\uff11\uff12\uff13\uff14\uff15\uff16\uff17\uff18\uff19";
+
+        public static string s_nonNumericsCodepoints = 
+            "abcdefghijklmnopqrstuvwxyz" +
+            "\u1369\u136a\u136b\u136c\u136d\u136e\u136f\u1370\u1371\u1372\u1373" + 
+            "\u1374\u1375\u1376\u1377\u1378\u1379\u137a\u137b\u137c\u137d";
+
+        public static string s_numericNonDecimalCodepoints =
+            "\u00b2\u00b3\u00b9\u1369\u136a\u136b\u136c\u136d\u136e\u136f\u1370" + 
+            "\u1371\u19da\u2070\u2074\u2075\u2076\u2077\u2078\u2079\u2080\u2081" + 
+            "\u2082\u2083\u2084\u2085\u2086\u2087\u2088\u2089\u2460\u2461\u2462" + 
+            "\u2463\u2464\u2465\u2466\u2467\u2468\u2474\u2475\u2476\u2477\u2478" + 
+            "\u2479\u247a\u247b\u247c\u2488\u2489\u248a\u248b\u248c\u248d\u248e" + 
+            "\u248f\u2490\u24ea\u24f5\u24f6\u24f7\u24f8\u24f9\u24fa\u24fb\u24fc" + 
+            "\u24fd\u24ff\u2776\u2777\u2778\u2779\u277a\u277b\u277c\u277d\u277e" +
+            "\u2780\u2781\u2782\u2783\u2784\u2785\u2786\u2787\u2788\u278a\u278b" +
+            "\u278c\u278d\u278e\u278f\u2790\u2791\u2792";
+
+        public static int [] s_numericNonDecimalValues = new int []
+        {   
+            2, 3, 1, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 0, 4, 5, 6, 7, 8, 9, 0, 1, 2, 
+            3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 
+            8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 0, 1, 
+            2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 
+            7, 8, 9
+        };
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public static void DigitsDecimalTest()
+        {
+            Assert.Equal(s_numericsCodepoints.Length % 10, 0);
+            for (int i=0; i<s_numericsCodepoints.Length; i+= 10)
+            {
+                for (int j=0; j<10; j++)
+                {
+                    Assert.Equal(CharUnicodeInfo.GetDecimalDigitValue(s_numericsCodepoints[i + j]), j);
+                    Assert.Equal(CharUnicodeInfo.GetDecimalDigitValue(s_numericsCodepoints, i + j), j);
+                } 
+            }
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public static void NegativeDigitsTest()
+        {
+            for (int i=0; i<s_nonNumericsCodepoints.Length; i++)
+            {
+                Assert.Equal(CharUnicodeInfo.GetDecimalDigitValue(s_nonNumericsCodepoints[i]), -1);
+                Assert.Equal(CharUnicodeInfo.GetDecimalDigitValue(s_nonNumericsCodepoints, i), -1);
+            }
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
```

Comment:
Yeah, as noted above, I'd suggest we switch these to be ActiveIssue.  That helps both by including the relevant issue number and by highlighting that it's disabled temporarily rather than permanently on that platform(s).